packages/astro/src/vite-plugin-astro/index.ts

```diff
@@ -196,20 +132,14 @@ export default function astro({ settings, logging }: AstroPluginOptions): vite.P
 				return;
 			}
 
-			const filename = normalizeFilename(parsedId.filename, config);
```

Comment:
In other diffs, you'll see that I had removed `normalizeFilename`. That's because `normalizeFilename` was previously used to emulate Vite's resolver algorithm, and only needed in a pre plugins `resolveId`.

Since this is a `transform` hook, we can remove it as it's already normalized by Vite.